pkg/kubelet/pleg/generic_test.go

```diff
@@ -363,7 +369,7 @@ func TestRelistWithCache(t *testing.T) {
 	statusErr := fmt.Errorf("unable to get status")
 	runtimeMock.EXPECT().GetPodStatus(ctx, pods[1].ID, "", "").Return(&kubecontainer.PodStatus{}, statusErr).Times(1)
 
-	pleg.Relist()
+	pleg.Relist(ctx)
```

Comment:
`ctx := context.Background()` on the line 353 above should be replaced with `tCtx := ktesting.Init(t)` I guess.